src/org/zaproxy/zap/spider/filters/ParseFilter.java

```diff
@@ -62,7 +62,7 @@ public FilterResult filtered(HttpMessage responseMessage) {
 	 *
 	 * @since 2.7.0
 	 * @see #NOT_FILTERED
-	 * @see #ParseFilter(String)
+	 * @see #filtered(HttpMessage)
```

Comment:
It should be `#FilterResult(String)` instead (though Java 8 does not process it correctly).